libutils/buffer.c

```diff
@@ -427,38 +427,6 @@ int BufferPrintf(Buffer *buffer, const char *format, ...)
     return printed;
 }
 
-int BufferVPrintf(Buffer *buffer, const char *format, va_list ap)
```

Comment:
Yes, but all printf-style functions suffer from this, and yet exploits are quite rare (usually `format` is a static string).